.travis.yml

```diff
@@ -8,11 +8,8 @@ env:
 
 matrix:
   include:
-    - python: 2.7
-    - python: 3.4
-      env: PILLOW="Pillow==2.9.0"
     - python: 3.5
-      env: PILLOW="Pillow==3.0.0"
+      env: PILLOW="Pillow==4.0.0"
```

Comment:
How about using the latest Pillow 5.0.0?

The main reason for the major version bump was dropping Python 3.3.

https://pillow.readthedocs.io/en/latest/releasenotes/5.0.0.html

Reply:
pillow 5.0 is in debian as well.